airbyte-integrations/connectors/source-callrail/source_callrail/manifest.yaml

```diff
@@ -26,7 +26,6 @@ definitions:
       datetime: "{{ today_utc() }}"
       datetime_format: "%Y-%m-%d"
     step: "#/definitions/step"
-    cursor_field: "{{ parameters.stream_cursor_field }}"
```

Comment:
This was a somewhat common case where cursor_field was different between streams, rather than rely on an interpolated reference to the stream’s `stream_cursor_field`, instead we can redefine the `$parameters` block to specify `cursor_field` which is then automatically propagated to the stream slicer when creating the components